docs/ru/hardaton_quidditch.md

```diff
@@ -0,0 +1,78 @@
+# Хардатон Квиддич.  
+
+## Название проекта: Инженерное командное соревнование - хардатон «Квиддич на квадрокоптерах».  
+
+![Название проекта](https://github.com/Ivan-Alekov/clover/blob/Hardaton_Quidditch/docs/assets/Hardaton_Quidditch/Hardaton.jpg)  
```

Comment:
External images are not allowed. Please, commit them to the repo.

(See in other articles how to embed them).